lib/resourcemerge/core_test.go

```diff
@@ -359,6 +359,100 @@ func TestEnsurePodSpec(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "modify container readiness probe",
+			existing: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name: "test",
+						ReadinessProbe: &corev1.Probe{
+							InitialDelaySeconds: 1,
+							TimeoutSeconds:      2,
+							PeriodSeconds:       3,
+							SuccessThreshold:    4,
+							FailureThreshold:    5,
+						},
+					},
+				},
+			},
+			input: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name: "test",
+						ReadinessProbe: &corev1.Probe{
+							InitialDelaySeconds: 7,
+							TimeoutSeconds:      8,
+							PeriodSeconds:       9,
+							SuccessThreshold:    10,
+							FailureThreshold:    11,
+						},
+					},
+				},
+			},
+
```

Comment:
nit: no need for a blank line here, or in the `modify container liveness probe` case below.  I don't see a blank line in master's `remove container ports` case above; did you have some precedent for this blank line?

Reply:
Unintentional, will fix up before we merge (if we want to merge). Will leave for now to avoid CI churn while we test

Reply:
Ì

Reply:
Done, thanks for reminding me, had forgotten